setup.cfg

```diff
@@ -102,12 +102,32 @@ formats = bdist_wheel
 
 [flake8]
 # Some sane defaults for the code style checker flake8
+ignore =
+    N805
+    F401 F811
+    W503
+    C901 # TODO: Complex function, (gmuraru) I think we should fix this
+per-file-ignores =
+    tests/**:DAR101, DAR201
+max-line-length = 100
```

Comment:
Q: I think in the black config we have 88? (Could you add the same values for all -- probably the same one that Syft uses)

Reply:
That is interesting. PySyft has a 120, as you told me remove the excludes section, I decided to return the max-line-length to the original that was in the .flake8rc configuration (100). Black has an 88 length line. You have the last word into what should I put: 120,100 or 88.

Reply:
Hmm..let's stick to the 100, could you also update the black?